src/components/BreadcrumbView/BreadcrumbView.tsx

```diff
@@ -78,13 +89,22 @@ export class BreadcrumbView extends React.Component<BreadCumbViewProps, BreadCum
     return this.state.pathItem === 'istio';
   };
 
+  isIter8 = () => {
+    return this.state.pathItem === 'iter8';
+  };
```

Comment:
This question is ambiguous in terms of time and space :-).
The answer is yes, it's documented:
https://kiali.io/documentation/latest/extensions/#_kiali_front_end_guideline

And also doc needs to be updated once things are consolidated.

For extensions, in the first approach, I tried to avoid tight couple with other shared components, but for these scenarios, perhaps I can create (in a follow up PR) some better extensibility model. I.e. a "ExtensionBreadcrumbView" which basically uses the "BreadcrumbView" but adding custom logic like this.

But I don't want to mix these concerns in the current PR as there are places that it still require explicit support (i.e. adding a new page).